app/front/scripts/services/download-package.js

```diff
@@ -80,8 +80,8 @@
               StorageService.clearApplicationState()
                   .then(function() {
                     var packageName = PackageService.getAttributes().name;
-                    var owner = LoginService.email;
-                    state.packagePublicUrl = '/viewer/'+owner+':'+packageName; //dataPackage.uploadUrl;
+                    var owner = LoginService.userid;
```

Comment:
um.....

but, we can't rely on the email staying stable with the auth provider either, hence, uuid :).

Example: GitHub as auth provider, and I change my github email address.